src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -1384,7 +1384,8 @@ private bool ShouldExecuteOperationBlockActions(AnalysisScope analysisScope, ISy
             var symbol = symbolEvent.Symbol;
             var executeSyntaxNodeActions = ShouldExecuteSyntaxNodeActions(analysisScope);
             var executeCodeBlockActions = ShouldExecuteCodeBlockActions(analysisScope, symbol);
-            var executeOperationActions = ShouldExecuteOperationActions(analysisScope);
+            // TODO: https://github.com/dotnet/roslyn/issues/9270
+            var executeOperationActions = this.analyzerExecutor.Compilation.IsIOperationFeatureEnabled() && ShouldExecuteOperationActions(analysisScope);
```

Comment:
Referencing #9270 in each location seems unnecessary since most also use `IsIOperationFeatureEnabled()`. Removing that method will uncover all these locations.